test/index.js

```diff
@@ -650,7 +651,7 @@ describe('mdast.process(value, options, done)', function () {
     it('should run plugins', function () {
         assert(
             mdast.use(mentions).process('@mention') ===
-            '[@mention](https://github.com/blog/821)\n'
+            '[@mention](https://github\\.com/blog/821)\n'
```

Comment:
This seems to break, at least in [commonmark](http://spec.commonmark.org/dingus/?text=%5B%40mention%5D(https%3A%2F%2Fgithub%5C%5C.com%2Fblog%2F821)), the URL. Maybe we need to differentiate between plain-text escapes from links, language-flags, reference/definition identifiers?